WalletWasabi.Documentation/Guides/UpgradeTorDaemons.md

```diff
@@ -0,0 +1,31 @@
+Check if there is a new Tor client. 
+
+Tor changelog here: https://gitweb.torproject.org/tor.git/plain/ChangeLog
+
+Download the latest stable Tor Browser from here: https://www.torproject.org/download/languages/
+
+- Windows x32
```

Comment:
Question: Why x32?

Reply:
Good catch, we were using x64 so far.

Reply:
Are you sure? Look at the current master https://github.com/zkSNACKs/WalletWasabi/tree/master/WalletWasabi/TorDaemons

Reply:
Had a discussion with @nopara73 IRL. Tor 32 was used because in the oldern days he used Tor Expert Bundle to extract the binaries and that was only x32. It is more consistent to extract this from the browser. I upgraded to x64.